ql/src/test/results/clientpositive/llap/mm_all.q.out

```diff
@@ -1649,8 +1649,6 @@ POSTHOOK: Lineage: ###Masked###
 POSTHOOK: Lineage: ###Masked###
 POSTHOOK: Lineage: ###Masked###
 POSTHOOK: Lineage: ###Masked###
-POSTHOOK: Lineage: ###Masked###
```

Comment:
Were these also duplicated rows? Could you please check the q.out.orig files before the masking?